.travis.yml

```diff
@@ -49,10 +49,14 @@ jobs:
         - pip install docker-ci-deploy==0.3.0
         - docker login -u "$REGISTRY_USER" -p "$REGISTRY_PASS"
       deploy:
-        provider: script
-        script: dcd --version "$(git rev-parse --short HEAD)" --version-latest "$IMAGE_NAME"
-        on:
-          branch: develop
+        - provider: script
+          script: dcd --version "$(git rev-parse --short HEAD)" --version-latest "$IMAGE_NAME"
+          on:
+            branch: develop
+        - provider: script
+          script: dcd --tag "$TRAVIS_TAG" --version "$(git rev-parse --short HEAD)" --version-latest "$IMAGE_NAME"
```

Comment:
I think I copy/pasted too much yesterday. With the git tag `mytag` pointing to commit `21fbcad`, this will produce the Docker tags:
```
praekeltfoundation/molo-gem:mytag
praekeltfoundation/molo-gem:21fbcad-mytag
```

You maybe don't want the git hash there. In which case you can drop the `--version` and `--version-latest` options.